dogen/generator.py

```diff
@@ -92,6 +92,9 @@ def configure(self):
             if os.path.exists(scripts) and os.path.isdir(scripts):
                 self.scripts_path = scripts
 
+        if not 'user' in self.cfg:
+            self.cfg['user'] = 0
+
```

Comment:
i would consider moving this to _validate_cfg just after yaml.safe_load() to have better readability - construct and adjust default of self.cfg in one place